editor/components/block-settings-menu/shared-block-convert-button.js

```diff
@@ -12,10 +12,20 @@ import { __ } from '@wordpress/i18n';
 import { isSharedBlock } from '@wordpress/blocks';
 import { withSelect, withDispatch } from '@wordpress/data';
 
-export function SharedBlockConvertButton( { sharedBlock, onConvertToStatic, onConvertToShared, itemsRole } ) {
+export function SharedBlockConvertButton( {
+	isVisible,
+	isStaticBlock,
+	onConvertToStatic,
+	onConvertToShared,
+	itemsRole,
+} ) {
```

Comment:
The only minor thing I would say is that my personal preference would be to move the destructuring to the body when props get quite long. But then I've noticed that's not really done much on this gutenberg, so I'm sure it's all good.

Reply:
Yeah, I'm totally just parroting what I've seen elsewhere 🙂